backend/api/schemas.py

```diff
@@ -0,0 +1,51 @@
+import json
+from pathlib import PosixPath
+from typing import Any
+
+import jsonschema
+from django.conf import settings
+
+from .exceptions import InvalidData
+
+
+class SchemaLoader:
+    _schema_dir: PosixPath = settings.BASE_DIR / "interfaces/schemas"
+    _schemas = set()
+    _cache = {}
+
+    def __init__(self):
+        self.reload()
+
+    def load(self, schema_name: str):
+        if not schema_name.endswith(".json"):
+            schema_name += ".json"
+        if schema_name not in self._schemas:
+            raise Exception(
+                "'%s' does not exists in folder '%s'. Consider reloading your "
+                "%s instance using the .reload() method"
+                % (schema_name, self._schema_dir, self.__class__.__name__)
```

Comment:
Is there a particular reason for you to use "`old style`" string formatting instead of `f-strings` ?

Reply:
Nope no reason.